manifests/f/FilesCommunity/Files/3.3.0.0/FilesCommunity.Files.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: FilesCommunity.Files
+PackageVersion: 3.3.0.0
+InstallerType: msix
+MinimumOSVersion: 10.0.19041.0
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.5
```

Comment:
```suggestion
  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.5
  - PackageIdentifier: Microsoft.VCLibs.140.00
```

Reply:
```
add-appxpackage : Deployment failed with HRESULT: 0x80073CF3, Package failed updates, dependency or conflict validation.
Windows cannot install package Files_3.3.0.0_x64__1y0xx7n9077q4 because this package depends on a framework that could not be found. Provide the
framework "Microsoft.VCLibs.140.00" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or
x64 processor architecture and minimum version 14.0.30704.0, along with this package to install.
NOTE: For additional information, look for [ActivityId] d450df79-93b8-000f-172c-51d4b893da01 in the Event Log or use the command line
Get-AppPackageLog -ActivityID d450df79-93b8-000f-172c-51d4b893da01
```

Reply:
It looks like @stephengillie provided a `PackageIdentifier` that doesn't exist - Try `Microsoft.VClibs.Desktop.14`

Reply:
@Trenly, I recognised that and changed it myself already, and it still does not work:

![image](https://github.com/microsoft/winget-pkgs/assets/81376423/b766747b-e5cf-47aa-8f9d-18dc5d688bb3)